bootstrap.sh

```diff
@@ -29,13 +29,13 @@ fi
 
 ################ parameters ################
 # directory where the script is located
-SCRIPT_DIR=$( cd $(dirname $0); pwd -P)
+SCRIPT_DIR="$( cd $(dirname $0); pwd -P)"
```

Comment:
but path shouldn't need to be wrapped in quotes?

I.e. from what I understand, quotes are parsed away, and `SCRIPT_DIR` gets all ths stuff from `$()`, and then later on all the contents of `SCRIPT_DIR` are being passed as a single argument to other stuff (since `SCRIPT_DIR` is enclosed in quotes whenever it is used.